Classes/LanguageManager.cpp

```diff
@@ -0,0 +1,91 @@
+#include "LanguageManager.h"
+
+LanguageManager* LanguageManager::_instance = 0;
+
+LanguageManager::LanguageManager()
+{
+	string fileName;
+	// detect current language
+	switch (CCApplication::getInstance()->getCurrentLanguage())
+	{
+	case LanguageType::ENGLISH:
```

Comment:
Currently our platform only supports English, Swahili, Spanish and Arabic. So those 4 languages would be sufficient.